src/main/kotlin/no/nav/tilleggsstonader/sak/behandlingsflyt/BehandlingSteg.kt

```diff
@@ -65,6 +65,11 @@ enum class StegType(
         tillattFor = BehandlerRolle.SYSTEM,
         gyldigIKombinasjonMedStatus = listOf(BehandlingStatus.IVERKSETTER_VEDTAK),
     ),
+    JOURNALFØRE_VEDTAKSBREV(
+        rekkefølge = 6,
+        tillattFor = BehandlerRolle.SYSTEM,
+        gyldigIKombinasjonMedStatus = listOf(BehandlingStatus.IVERKSETTER_VEDTAK),
+    ),
```

Comment:
@blommish Nå har jeg lagt opp til at det blir et eget steg for journalføring, og et for distribuering. Men man kunne evt. bare hatt det i et steg, men med flere tasker på det steget. 

Tanker? Kanskje greit å ha en en-til-en kobling mellom taskene og stegene 🤔